docs/reference/announcements/880.md

```diff
@@ -137,25 +137,20 @@ Additional upgrade considerations are necessary for deployments that use custom
 
 ## Camunda Exporter <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
 
-A new Camunda Exporter brings the importer and archiving logic of web components (Tasklist and Operate) closer to the distributed platform (Zeebe). The index schema is also being harmonized.
+Our web applications have historically used importers and archivers to consume, aggregate, and archive historical data provided by our Elasticsearch (ES) or OpenSearch (OS) Exporter.
```

Comment:
```suggestion
Our web applications have historically used importers and archivers to consume, aggregate, and archive historical data provided by our Elasticsearch (ES) or OpenSearch (OS) Exporters.
```
Slightly ashamed as a native speaker that I am not fully sure if my suggestion is correct here. Maybe the tech writers can confirm 🙈